technology/developing-appchain.md

```diff
@@ -4,32 +4,33 @@ Launching an Appchain with Pelagos is as simple as deploying a smart contract. T
 
 - Hash of a Docker container
 - (Optional) genesis data
-- Info-hash (for immutable database distribution)
+- (Optional) Info-hash (for immutable database distribution)
 
 and run each Appchain exactly as one runs one, or multiple, microservices.
 
+<!-- Whats the ask here: not really related things: "With Pelagos, these gRPC setup steps are hidden behind simple callback hooks and event subscriptions, allowing developers to focus on business logic." 
+ -->
+
 Normally, using gRPC in a distributed system requires developers to define services in Protocol Buffers (Protobuf), handle client and server pairing and stub generation, HTTP/2 multiplexing, and deal with stream or unary message handling, serialization, and error management. With Pelagos, these gRPC setup steps are hidden behind simple callback hooks and event subscriptions, allowing developers to focus on business logic.
```

Comment:
gRPC is unrelated to the hooks function --
As dev must specify how to handle user transactions, 

Abstracted away: Security layer, sequencing of transactions, accessing external chain data, discovering rates, accessing external liquidity pools
All they have to do is interact with contracts, optionally reactive contracts and determine how to handle user transactions could leverage your own API (or might use an existing gRPC or use alt logic) -- > 
Drip the whole challenge bit 
Normally, using gRPC in a distributed system requires developers to define services in Protocol Buffers (Protobuf), handle client and server pairing and stub generation, HTTP/2 multiplexing, and deal with stream or unary message handling, serialization, and error management.

Reply:
moved inline